src/main/java/domain/Notes.java

```diff
@@ -0,0 +1,70 @@
+package domain;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "notes")
+public class Notes {
+
```

Comment:
No, because other classes aren't named correctly and later we need to do refractor.